src/main/java/com/gibraltar/iberia/challenge/NavigationChallenge.java

```diff
@@ -39,18 +35,12 @@
 import net.minecraftforge.fml.relauncher.Side;
 import net.minecraftforge.fml.relauncher.SideOnly;
 
-import com.google.gson.Gson;
-import com.google.gson.JsonElement;
-import com.google.gson.JsonObject;
-
 import com.gibraltar.iberia.items.ItemPersonalCompass;
 import com.gibraltar.iberia.challenge.SpawnChallenge;
-import com.gibraltar.iberia.network.MessageRegistry;
-import com.gibraltar.iberia.network.MessageGetPlayerSpawn;
 import com.gibraltar.iberia.Reference;
 
 public class NavigationChallenge extends Challenge {	
```

Comment:
My first test at compasses shows they are not pointing the right direction, not sure why yet.

Reply:
Ok, compasses obtained by /give aren't working, but crafting them seems to work. Not sure if that's a bug in 1.11 too.

Reply:
since its meant to replace the vanilla compass, I removed that recipe, which is the only way to make it craftable with the same recipe. unfortunately this messes with the crafting achievement for the compass. no way to fix that as far as I know

Reply:
you were crafting the minecraft compass

Reply:
You're right though, the compass doesn't work, but I'm not sure why, I haven't missed with compasses so you'll need to debug it